src/test/java/lambda/part1/exercise/Exercise1.java

```diff
@@ -60,8 +66,10 @@ public void sortPersonsByLastNameThenFirstNameUsingArraysSortAnonymousComparator
         Arrays.sort(persons, new Comparator<Person>() {
             @Override
             public int compare(Person o1, Person o2) {
-
-                return o1.getLastName().compareTo(o2.getLastName())!=0?o1.getLastName().compareTo(o2.getLastName()):o1.getFirstName().compareTo(o2.getFirstName());
+                boolean notEquals = o1.getLastName().compareTo(o2.getLastName())!=0;
```

Comment:
There is lifehack: if you calculate `o1.getLastName().compareTo(o2.getLastName())` once - you can use it's result at line 71.

But still, good job!